fgcm/fgcmLUT.py

```diff
@@ -282,7 +282,7 @@ def makeLUT(self):
         self.atmStdTrans = self.atmosphereTable.atmStdTrans
 
         self.pmbElevation = self.atmosphereTable.pmbElevation
-
+        
```

Comment:
???

Reply:
Oops, well, it's going to sit there since patching it is a pain.